### PR TITLE
269 final code cleanup for public release

### PR DESCRIPTION
**Pull request by Jacob1010-h**

### CODE REVIEW

File: src/main/java/frc/robot/RobotState.java
[the remark this answers is not in the document]
Comment (Jacob1010-h) [reply src/main/java/frc/robot/RobotState.java:1]: ? oh i didnt know they did that, that was just the first thing that I thought of. Ill rename it.

**Comment by Jacob1010-h** [comment]
I agree, if we end up going to Bayou or worlds, then we should wait.

**Comment by Jacob1010-h** [comment]
does that also mean that you wouldn't want to make this public, cause if so, we could probably wait until after then too to make it public

File: src/main/java/frc/robot/RobotState.java
[the remark this answers is not in the document]
Comment (GalexY727) [reply src/main/java/frc/robot/RobotState.java:1]: All good, that original comment was very.... aggressive. I apologize for that tone.

File: src/main/java/frc/robot/util/calc/ShooterCalc.java
Comment (GalexY727) [inline src/main/java/frc/robot/util/calc/ShooterCalc.java:1]: I in fact never tested this and would still like to keep it in todos, for future reference.

Comment (Jacob1010-h) [reply src/main/java/frc/robot/util/calc/ShooterCalc.java:1]: Everything on this branch has been tested in sim, however it's possible I overlooked something and I would like to actually test it on the bot when we plan on merging this branch into main.

**Comment by Jacob1010-h** [comment]
I am going to reopen this and make this a draft, because we can do that now that the repo is public

**Comment by GalexY727** [comment]
tbh, i think that the folder paths in main are good, thinking about it further. When I think of a file, the only path to get there would be to follow the folders, and the folder names explain the purpose of everything inside of it which is how it is supposed to be

**Comment by Jacob1010-h** [comment]
> tbh, i think that the folder paths in main are good, thinking about it further. When I think of a file, the only path to get there would be to follow the folders, and the folder names explain the purpose of everything inside of it which is how it is supposed to be

So you think that the folder paths we have in main area the better option?

**Comment by GalexY727** [comment]
yes

**Pull request closed without merging**